### ISSUE TITLE
JoinLanding content not wrapping on mobile

### ISSUE THREAD

**Issue by ryancreatescopy** [ethereum/clrfund#89]
![image](https://user-images.githubusercontent.com/40891631/119642727-ef651500-be12-11eb-8b13-7a397f238fb4.png)

This may only affect the `RoundClosed` state

**Comment by samajammin** [comment]
Whoever picks this up: please confirm this is fixed for each round state.

Also worth investigating that all landing pages don't have this same issue...

**Comment by pettinarip** [comment]
I've tested the following landing pages:
- Join: I couldn't reproduce the issue described here. I've tested in different resolutions, not only mobile, in different round states and the content is wrapping correctly. (however, I've found a small issue while testing => fixed in #181)
- Join/success: working correctly.
- Verify: working correctly.
- Verify/success: working correctly.
- Home: working correctly.
